docker/docker.go

```diff
@@ -161,6 +161,12 @@ func (d *ImageManager) FindImage(imageName string) (*dockerclient.Image, error)
 	return image, nil
 }
 
+// HasImage determines if the given image already exists in Docker
+func (d *ImageManager) HasImage(imageName string) bool {
+	_, err := d.FindImage(imageName)
+	return (err == nil)
```

Comment:
Addressed in 93e35bbfdeafbff5f32116e3b0982885593d9f8f

Also, it turns out that the [current code](https://github.com/hpcloud/fissile/blob/afffe8af344c4fc33557b722df5dfa8177b7bdb4/app/fissile.go#L371) also throws away the error (but at least it assumes we _will_ need to rebuild).